test/cypress/features/blocks/blocks.js

```diff
@@ -0,0 +1,70 @@
+/* eslint-disable */ 
+import { Then } from 'cypress-cucumber-preprocessor/steps';
+import { ss } from '../../../constants';
+
+let oldHeight, currentHeight
+
+Then(/^I should see (\d+) blocks in table$/, function (number) {
+  cy.wait(500);
```

Comment:
I think instead of putting the wait command here, it can be defined in the feature file just before this test case is defined something like 

`When I wait 0.5 seconds`

Reply:
@eniolam1000752 A lot of the feature tests are set up like this because it prevents verbosity when waiting for a very short period (< 1 second) before proceeding with a test. Therefore writing things like `When I wait for 0.5 seconds` or `When I wait for 0.1 seconds` unnecessarily adds more lines to the tests. In cases, where a common action like clicking on a button has a minor wait time too, writing the extra line in all the features where an element is clicked would be a lot of work without a lot of obvious value